htdocs/index.php

```diff
@@ -448,7 +448,7 @@
 }
 
 // Number of supplier invoices (has paid)
-if (! empty($conf->supplier_invoice->enabled) && ! empty($conf->facture->enabled) && $user->rights->facture->lire)
+if (! empty($conf->supplier_invoice->enabled) && ! empty($conf->facture->enabled) && $user->rights->fournisseur->facture->lire)
```

Comment:
@joseplluis 

$conf->facture->enabled is not for this module !

replace with :
if (! empty($conf->supplier_invoice->enabled) && ! empty($user->rights->fournisseur->facture->lire))